{{Lund}}
<html>
</div> <!-- iGEM content -->

<script type="text/javascript">
  $('#navbar').addClass('navbar-transparent');
</script>

<header class="experiments-landing">
  <div class="container">
    <h1 class="display">Experiments</h1>
  </div>
</header>

<div class="content-wrapper">
  <div class="container">
    <section id="experiments">
      <div class="well experiment-container">
        <a data-toggle="collapse" data-parent="#experiments" href="#agarplates">
          <h5>
            <i class="fa fa-flask" aria-hidden="true"></i>
             Preparation of agar plates
           </h5>
        </a>
        <div id="agarplates" class="collapse">
          <div class="row">
            <div class="col-md-6">
              <u>Material</u>
              <ul>
                <li>500 mL dH2O</li>
                <li>12 g Agar agar</li>
              </ul>
            </div>
            <div class="col-md-6">
              <u>Procedure</u>
              <ol>
                <li>Add 250 mL dH2O to beaker</li>
                <li>Add all dry reagents</li>
                <li>Add another 250 mL dH2O, stir and autoclave</li>
              </ol>
            </div>
          </div>
        </div>
      </div>

      <div class="well experiment-container">
        <a data-toggle="collapse" data-parent="#experiments" href="#lbmedium">
          <h5>
            <i class="fa fa-flask" aria-hidden="true"></i>
             Preparation of LB medium
           </h5>
        </a>
        <div id="lbmedium" class="collapse">
          <div class="row">
            <div class="col-md-6">
              <u>Material</u>
              <ul>
                <li>500 mL dH2O</li>
                <li>12 g Agar agar</li>
              </ul>
            </div>
            <div class="col-md-6">
              <u>Procedure</u>
              <ol>
                <li>Add 250 mL dH2O to beaker</li>
                <li>Add all dry reagents</li>
                <li>Add another 250 mL dH2O, stir and autoclave</li>
              </ol>
            </div>
          </div>
        </div>
      </div>

      <div class="well experiment-container">
        <a data-toggle="collapse" data-parent="#experiments" href="#plasmidisolation">
          <h5>
            <i class="fa fa-flask" aria-hidden="true"></i>
            Plasmid isolation
           </h5>
        </a>
        <div id="plasmidisolation" class="collapse">
          <div class="row">
            <div class="col-md-6">
              <u>Material</u>
              <ul>
                <li>NucleoSpin® Plasmid (NoLid) Plasmid DNA purification Kit</li>
                <li>Sterile MQ water</li>
                <li>Cell culture</li>
              </ul>
              <u>Equipment</u>
              <ul>
                <li>Centrifuge</li>
                <li>Vortex (optionally)</li>
                <li>Lab timer</li>
                <li>Pipettes and Tips (1000 µL and 500 µL recommended)</li>
                <li>1.5 mL microcentrifuge tubes</li>
                <li>Nanophotometer</li>
              </ul>
            </div>
            <div class="col-md-6">
              <u>Procedure</u>
              <ol>
                <li>Add 1 mL of Buffer A1 to the RNase A vial and vortex. Transfer the solution back into the Buffer A1 bottle and mix thoroughly. Indicate date of RNase A addition. Store Buffer A1 containing RNase A at 4 °C. The solution will be stable at this temperature for at least six months.</li>
                <li>Add 100 mL of 96–100 % ethanol to Buffer A4.</li>
                <li><b>Cultivate and harvest bacterial cells.</b> Use 1–5 mL of a saturated E.coli LB culture, pellet cells in a standard benchtop microcentrifuge for 30 s at 11,000 x g. Discard the supernatant and remove as much of the liquid as possible.</li>
                <li><b>Cell lysis.</b> Add 250 μL Buffer A1. Resuspend the cell pellet completely by vortexing or pipetting up and down. Make sure no cell clumps remain before addition of Buffer A2!</li>
                <li>Add 250 μL Buffer A2. Mix gently by inverting the tube 6–8 times. Do not vortex to avoid shearing of genomic DNA. Incubate at room temperature for up to 5 min or until lysate appears clear. </li>
                <li>Add 300 μL Buffer A3. Mix thoroughly by inverting the tube 6–8 times until blue samples turn colorless completely! Do not vortex to avoid shearing of genomic DNA!</li>
                <li>Make sure to neutralize completely to precipitate all protein and chromosomal DNA. LyseControl should turn completely colorless without any traces of blue.</li>
                <li><b>Clarification of lysate.</b> Centrifuge for 5 min at 11,000 x g at room temperature. <br>Repeat this step in case the supernatant is not clear!</li>
                <li><b>Bind DNA.</b> Place a NucleoSpin® Plasmid/Plasmid (NoLid) Column in a Collection Tube (2 mL) and decant the supernatant from step 3 or pipette a maximum of 750 μL of the supernatant onto the column. Centrifuge for 1 min at 11,000 x g. Discard flow-through and place the NucleoSpin® Plasmid/Plasmid (NoLid) Column back into the collection tube. Repeat this step to load the remaining lysate.</li>
                <li><b>Wash silica membrane.</b> Add 600 μL Buffer A4 (supplemented with ethanol, see section 3). Centrifuge for 1 min at 11,000 x g. Discard flow-through and place the NucleoSpin® Plasmid / Plasmid (NoLid) Column back into the empty collection tube.</li>
                <li><b>Dry silica membrane.</b> Centrifuge for 2 min at 11,000 x g and discard the collection tube. Note: Residual ethanolic wash buffer might inhibit enzymatic reactions.</li>
                <li><b>Elute DNA.</b> Place the NucleoSpin® Plasmid / Plasmid (NoLid) Column in a 1.5 mL microcentrifuge tube (not provided) and add 50 μL Buffer AE. Incubate for 1 min at room temperature. Centrifuge for 1 min at 11,000 x g.</li>
                <li>Measure the concentration of DNA using nanophotometer. </li>
              </ol>
              Source: <a href="http://www.mn-net.com/Portals/8/attachments/Redakteure_Bio/Protocols/Plasmid%20DNA%20Purification/UM_pDNA_NS.pdf ">Macherey-Nagel Plasmid DNA purification Manual </a>
            </div>
          </div>
        </div>
      </div>

      <div class="well experiment-container">
        <a data-toggle="collapse" data-parent="#experiments" href="#transformation">
          <h5>
            <i class="fa fa-flask" aria-hidden="true"></i>
            Transformation
           </h5>
        </a>
        <div id="transformation" class="collapse">
          <div class="row">
            <div class="col-md-6">
              <u>Material</u>
              <ul>
                <li>Resuspended DNA to be transformed</li>
                <li>10 pg/µL positive transformation control DNA (e.g. pSB1C3 w/ BBa_J04450, RFP on high-copy chloramphenicol resistant plasmid. Located in the Competent Cell Test Kit.)</li>
                <li>Competent cells (50 µL per sample)</li>
                <li>1.5 mL microtubes</li>
                <li>SOC media (950 µL per sample)</li>
                <li>Petri plates w/ LB agar and antibiotic (2 per sample)</li>
              </ul>
              <u>Equipment</u>
              <ul>
                <li>Ice and ice bucket</li>
                <li>Lab Timer</li>
                <li>42°C water bath</li>
                <li>37°C incubator</li>
                <li>Sterile spreader</li>
                <li>Pipettes and tips (10 µL, 20µl, 200 µL, 1000 µL recommended)</li>
                <li>Microcentrifuge</li>
              </ul>
            </div>
            <div class="col-md-6">
              <u>Procedure</u>
              <ol>
                <li>Resuspend DNA in selected wells in the Distribution Kit with 10 µL dH20. Pipet up and down several times, let sit for a few minutes. Resuspension will be red from cresol red dye.</li>
                <li>Label 1.5 mL tubes with part name or well location. Fill lab ice bucket with ice, and pre-chill 1.5 mL tubes (one tube for each transformation, including your control) in a floating foam tube rack.</li>
                <li><b>Thaw competent cells on ice:</b> This may take 10-15 min for a 260 µL stock. Dispose of unused competent cells. Do not refreeze unused thawed cells, as it will drastically reduce transformation efficiency.</li>
                <li><b>Pipette 50 µL of competent cells into 1.5 mL tube:</b> 50 µl in a 1.5 mL tube per transformation. Tubes should be labeled, pre-chilled, and in a floating tube rack for support. Keep all tubes on ice. Don’t forget a 1.5 mL tube for your control.</li>
                <li><b>Pipette 1 µL of resuspended DNA into 1.5mL tube:</b> Pipette from well into appropriately labeled tube. Gently pipette up and down a few times. Keep all tubes on ice.</li>
              </ol>
            </div>
          </div>
        </div>
      </div>

      <div class="well experiment-container">
        <a data-toggle="collapse" data-parent="#experiments" href="#transformationquick">
          <h5>
            <i class="fa fa-flask" aria-hidden="true"></i>
            Transformation - quick protocol
           </h5>
        </a>
        <div id="transformationquick" class="collapse">
          <div class="row">
            <div class="col-md-6">
              <u>Material</u>
              <ul>
                <li>Resuspended DNA to be transformed</li>
                <li>10 pg/µl positive transformation control DNA (e.g. pSB1C3 w/ BBa_J04450, RFP on high-copy chloramphenicol resistant plasmid. Located in the Competent Cell Test Kit.)</li>
                <li>Competent cells (50µl per sample)</li>
                <li>1.5 mL microtubes</li>
                <li>SOC Media (950µL per sample)</li>
                <li>Petri plates with LB agar and antibiotic (2 per sample)</li>
              </ul>
              <u>Equipment</u>
              <ul>
                <li>Ice and ice bucket</li>
                <li>Lab timer</li>
                <li>37 °C dry bath/heat block</li>
                <li>37 °C incubator</li>
                <li>Sterile spreader</li>
                <li>Pipettes and tips (10µl, 20µl, 200µl, 1000µl recommended)</li>
                <li>Microcentrifuge</li>
              </ul>
            </div>
            <div class="col-md-6">
              <u>Procedure</u>
              <ol>
                <li>Resuspend DNA in selected wells in the Distribution Kit with 10 µL dH20. Pipet up and down several times, let sit for a few minutes. Resuspension will be red from cresol red dye.</li>
                <li>Label 1.5 mL tubes with part name or well location. Fill lab ice bucket with ice, and pre-chill 1.5 mL tubes (one tube for each transformation, including your control) in a floating foam tube rack. Pre-heat the agar plates by putting them in the 37 °C incubator.</li>
                <li><b>Thaw competent cells on ice:</b> This may take 10-15 min for a 260 µL stock. Dispose of unused competent cells. Do not refreeze unused thawed cells, as it will drastically reduce transformation efficiency.</li>
                <li><b>Pipette 50 µL of competent cells into 1.5 mL tube:</b> 50 µL in a 1.5 mL tube per transformation. Tubes should be labeled, pre-chilled, and in a floating tube rack for support. Keep all tubes on ice. Don’t forget a 1.5 mL tube for your control.</li>
                <li><b>Pipette 1 µL of resuspended DNA into 1.5 mL tube:</b> Pipette from well into appropriately labeled tube. Tap the tube gently to mix. Keep all tubes on ice.</li>
              </ol>
            </div>
          </div>
        </div>
      </div>

      <div class="well experiment-container">
        <a data-toggle="collapse" data-parent="#experiments" href="#digestionandligation">
          <h5>
            <i class="fa fa-flask" aria-hidden="true"></i>
            Digestion and ligation
           </h5>
        </a>
        <div id="digestionandligation" class="collapse">
          <div class="row">
            <div class="col-md-6">
              <u>Material</u>
              <ul>
                <li>Linearised plasmid backbone pSB1C3 DNA to be transformed</li>
                <li>gBlock fragments (25 ng/&micro;L)</li>
                <li>10 pg/µL positive transformation control DNA (e.g. pSB1C3 w/ BBa_J04450, RFP on high-copy chloramphenicol resistant plasmid. Located in the Competent Cell Test Kit.)</li>
                <li>Competent cells (50 µL per sample)</li>
                <li>1.5 mL microtubes</li>
                <li>SOC media (200 µL per sample)</li>
                <li>Petri plates w/ LB agar and antibiotic (2 per sample)</li>
              </ul>
              <u>Equipment</u>
              <ul>
                <li>Ice and ice bucket</li>
                <li>Lab timer</li>
                <li>80 °C incubator</li>
                <li>37 °C incubator</li>
                <li>Sterile spreader </li>
                <li>Pipettes and Tips (2 µL, 10 µL, 20 µL, 200 µL recommended)</li>
                <li>Microcentrifuge</li>
              </ul>
            </div>
            <div class="col-md-6">
              <u>Procedure digest</u>
              <ol>
                  <li>Enzyme Master Mix for Plasmid Backbone (25 uL total, for 5 rxns)</li>
                  <ul>
                    <li>5 &micro;L NEB Buffer 2</li>
                    <li>0.5 &micro;L EcoRI-HF</li>
                    <li>0.5 &micro;L PstI</li>
                    <li>19 &micro;L dH20</li>
                  </ul>
                  <li>Digest Plasmid Backbone</li>
                  <ul>
                    <li>Add 4 &micro;L linearized plasmid backbone (25 ng/&micro;L for 100 ng total)</li>
                    <li>Add 4 &micro;L of Enzyme Master Mix</li>
                    <li>Digest 37C/30 min, heat kill 80C/20 min</li>
                  </ul>
              </ol>
              <u>Procedure ligation</u>
              <ol>
                <li>Add 2&micro;L of digested plasmid backbone (25 ng)</li>
                <li>Add equimolar amount of EcoRI-HF PstI digested fragment (6 &micro;L)</li>
                <li>Add 1 &micro;L T4 DNA ligase buffer. Note: Do not use quick ligase</li>
                <li>Add 0.5 &micro;L T4 DNA ligase</li>
                <li>Add water to 10 &micro;L</li>
                <li>Ligate 16C/30 min, heat kill 80C/20 min</li>
                <li>Transform with 1-2 &micro;L of product</li>
              </ol>
            </div>
          </div>
        </div>
      </div>

      <div class="well experiment-container">
        <a data-toggle="collapse" data-parent="#experiments" href="#gelelectro">
          <h5>
            <i class="fa fa-flask" aria-hidden="true"></i>
            Agarose gel electrophoresis
           </h5>
        </a>
        <div id="gelelectro" class="collapse">
          <div class="row">
            <div class="col-md-6">
              <u>Material</u>
              <ul>
                <li>70% ethanol</li>
                <li>TBE Buffer</li>
                <li>Agarose powder</li>
                <li>Samples</li>
                <li>Loading dye</li>
                <li>Molecular Ladder</li>
                <li>Gel Red</li>
              </ul>
              <u>Equipment</u>
              <ul>
                <li>Microwave</li>
                <li>Pipettor</li>
                <li>Pipette tips</li>
                <li>Gel Box with Gel Tray</li>
                <li>Microwaveable Flask</li>
                <li>Measuring Glass</li>
                <li>Scale</li>
                <li>Weighing ship</li>
                <li>Electrodes and Power Source</li>
              </ul>
            </div>
            <div class="col-md-6">
              <u>Procedure</u>
              <ol>
                  <li>Clean your working area by wiping down with 70% ethanol.</li>
                  <li>Weigh up 0.5g of agarose in a weighing ship for a 1% agarose gel.</li>
                  <li>Measure up 50mL of TBE buffer in a measuring glass.
                  <b>Note:</b> Make sure to use the same buffer as the one in the gel box (do not mix different buffers and do not use water).</li>
                  <li>Add the TBE and the agarose powder onto a microwaveable flask and swirl to mix.</li>
                  <li>Microwave the mixture for 1-3 min and check for clarity and complete dissolvation while swirling.
                  <b>Note:</b> Do not overboil the solution, as some of the buffer will evaporate and thus alter the final percentage of agarose in the gel. Many people prefer to microwave in pulses, swirling the flask occasionally as the solution heats up.</li>
                  <li>Once clear, let the agarose cool cown for some time (to not damage the gel trays), approximately 5 min.</li>
                  <li>Add 5uL of Gel Red. Swirl to mix. </li>
                  <li>Pour the agarose carefully onto the gel try with the well comb well attached and in place.
                  <b>Note:</b> Pour slowly to avoid bubbles which will disrupt the gel. Any bubbles can be pushed away from the well comb or towards the sides/edges of the gel with a pipette tip.</li>
                  <li>Let it solidify for approximately 30 min.</li>
                  <li>Remove the well comb from the solid gel and place it in the gel box (electrophoresis unit).</li>
                  <li>Fill up the gel box with running buffer to the maximum marking, or until the gel is completely covered.</li>
                  <li>Prepare the samples with loading dye.
                  <b>Note:</b> Loading dye serves two purposes: 1) it provides a visible dye that helps with gel loading and will also allows you to gauge how far the gel has run while you are running your gel; and 2) it contains a high percentage of glycerol, so it increases the density of your DNA sample causing it settle to the bottom of the gel well, instead of diffusing in the buffer.</li>
                  <li>Pipett a molecular ladded onto the first well and then continue with adding the samples onto the following wells.
                  <b>Note:</b> When loading the sample in the well, maintain positive pressure on the sample to prevent bubbles or buffer from entering the tip. Place the very top of the tip of the pipette into the buffer just above the well. Very slowly and steadily, push the sample out and watch as the sample fills the well. After all of the sample is unloaded, push the pipettor to the second stop and carefully raise the pipette straight out of the buffer.</li>
                  <li>Run the gel at 80-150 V until the dye line is approximately 75-80% of the way down the gel.
                  <b>Note:</b> Black is negative, red is positive. (The DNA is negatively charged and will run towards the positive electrode.) Always Run to Red.</li>
                  <li>Turn OFF power, disconnect the electrodes from the power source, and then carefully remove the gel from the gel box.</li>
                  <li>Use any device that has UV light to visualize the DNA fragments on the gel.
                  <b>Note:</b> If you will be purifying the DNA for later use, use long-wavelength UV and expose for as little time as possible to minimize damage to the DNA.
                  <b>Note:</b> The fragments of DNA are usually referred to as 'bands’ due to their appearance on the gel.</li>
                </ol>
                source: <a href="https://www.addgene.org/protocols/gel-electrophoresis/ ">addgene</a>
            </div>
          </div>
        </div>
      </div>

      <div class="well experiment-container">
        <a data-toggle="collapse" data-parent="#experiments" href="#agarosednaextraction">
          <h5>
            <i class="fa fa-flask" aria-hidden="true"></i>
            DNA extraction from agarose gels
           </h5>
        </a>
        <div id="agarosednaextraction" class="collapse">
          <div class="row">
            <div class="col-md-6">
              <u>Material</u>
              <ul>
                <li>NucleoSpin® Gel and PCR Clean-up Kit</li>
                <li>Sterile MQ water</li>
                <li>agarose gel with the DNA sample to be extracted</li>
              </ul>
              <u>Equipment</u>
              <ul>
                <li>Centrifuge</li>
                <li>Scalpel</li>
                <li>Vortex</li>
                <li>Lab timer</li>
                <li>Balance</li>
                <li>Pipettes and Tips (1000 µL and 500 µL recommended)</li>
                <li>1.5 mL microcentrifuge tubes</li>
                <li>Nanophotometer</li>
              </ul>
            </div>
            <div class="col-md-6">
              <u>Procedure</u>
              <ol>
                <li><b>Excise DNA fragment / solubilize gel slice.</b> Take a clean scalpel to excise the DNA fragment from an agarose gel. Remove all excess agarose. Determine the weight of the gel slice and transfer it to a clean tube. </li>
                <li>For each 100 mg of agarose gel <  2 % add 200 μL Buffer NTI. For gels containing > 2 % agarose, double the volume of Buffer NTI. Incubate sample for 5–10 min at 50 °C. </li>
                <li>Vortex the sample briefly every 2–3 min until the gel slice is completely dissolved!</li>
                <li><b>Bind DNA.</b> Place a NucleoSpin® Gel and PCR Clean-up Column into a Collection Tube (2 mL) and load up to 700  μL sample. Centrifuge for 30 s at 11,000 x g. Discard flow-through and place the column back into the collection tube. Load remaining sample if necessary and repeat the centrifugation step.</li>
                <li><b>Wash silica membrane.</b> Add 700 μL Buffer NT3 to the NucleoSpin® Gel  and PCR Clean-up Column. Centrifuge for 30 s at 11,000 x g. Discard flow-through and place the column back into the collection tube. Recommended: Repeat previous washing step to minimize chaotropic salt carry-over and low A260/A230</li>
                <li><b>Dry silica membrane.</b> Centrifuge for 1 min at 11,000 x g to remove Buffer NT3 completely. Make sure the spin column does not come in contact with the flow-through while removing it from the centrifuge and the collection tube. Note: Residual ethanol from Buffer NT3 might inhibit enzymatic reactions. Total removal of ethanol can be achieved by incubating the columns for 2–5 min at 70 °C prior to elution.</li>
                <li><b>Elute DNA.</b> Place the NucleoSpin® Gel and PCR Clean-up Column into a new 1.5 mL microcentrifuge tube (not provided). Add 15–30 μL Buffer NE and incubate at room temperature (18–25 °C) for 1 min. Centrifuge for 1 min at 11,000 x g.</li>
                <li>Measure the DNA concentration using nanophotometer</li>
                <li></li>
              </ol>
              <a href="http://www.mn-net.com/Portals/8/attachments/Redakteure_Bio/Protocols/DNA%20clean-up/UM_PCRcleanup_Gelex_NSGelPCR.pdf"> Macherey-Nagel NucleoSpin® Gel and PCR Clean-up Manual</a>
            </div>
          </div>
        </div>
      </div>

      <div class="well experiment-container">
        <a data-toggle="collapse" data-parent="#experiments" href="#cellprep">
          <h5>
            <i class="fa fa-flask" aria-hidden="true"></i>
            Preparation of competent <i>E. coli</i> cells
           </h5>
        </a>
        <div id="cellprep" class="collapse">
          <div class="row">
            <div class="col-md-6">
              <u>Material</u>
              <ul>
                <li>500 mL dH2O</li>
                <li>12 g Agar agar</li>
              </ul>
            </div>
            <div class="col-md-6">
              <u>Procedure</u>
              <ol>
                <li>Add 250 mL dH2O to beaker</li>
                <li>Add all dry reagents</li>
                <li>Add another 250 mL dH2O, stir and autoclave</li>
              </ol>
            </div>
          </div>
        </div>
      </div>

      <div class="well experiment-container">
        <a data-toggle="collapse" data-parent="#experiments" href="#pcr">
          <h5>
            <i class="fa fa-flask" aria-hidden="true"></i>
            PCR
           </h5>
        </a>
        <div id="pcr" class="collapse">
          <div class="row">
            <div class="col-md-6">
              <u>Material</u>
              <ul>
                <li>70% ethanol</li>
                <li>Milliq Water</li>
                <li>Q5 High-Fidelity 2X Master Mix</li>
                <li>10uM Forward Primer</li>
                <li>10uM Reverse Primer</li>
                <li>Template DNA</li>
              </ul>
              <u>Equipment</u>
              <ul>
                <li>PCR Machine</li>
                <li>Pipettor</li>
                <li>Pipette tips</li>
                <li>Vortexer</li>
                <li>Table Top Centrifuge</li>
                <li>PCR Tubes</li>
              </ul>
            </div>
            <div class="col-md-6">
              <u>Procedure</u>
              <ol>
                <li>Clean your working area by wiping down with 70% ethanol.</li>
                <li>Label one PCR tube for each desired reaction. </li>
                <li>Prepare a PCR reaction for each colony. Easiest is to prepare a Master Mix with Q5 High-Fidelity 2X Master Mix, Milliq Water and possibly primers and then aliquot into the reaction tubes. Start by adding the lowest amount (the template DNA).
                  <br>
                  <br>
                  <table style="">
                   <tr style="border-bottom: 1px solid black">
                     <th>Component</th>
                     <th>12.5 uL reaction</th>
                     <th>25 uL reaction</th>
                     <th>50uL reaction</th>
                     <th>Final Concentration</th>
                   </tr>
                   <tr>
                     <td>Q5 High-Fidelity 2X Master Mix</td>
                     <td>6.25 uL</td>
                     <td>12.5 uL</td>
                     <td>25 uL</td>
                     <td>1X</td>
                   </tr>
                   <tr>
                    <td>10uM Forward Primer</td>
                    <td>0.75 uL</td>
                    <td>1.25uL</td>
                    <td>2.5 uL</td>
                    <td>0.5 uM</td>
                   </tr>
                   <tr>
                    <td>10uM Reverse Primer</td>
                    <td>0.75 uL</td>
                    <td>1.25uL</td>
                    <td>2.5 uL</td>
                    <td>0.5 uM</td>
                  </tr>
                    <td>Milliq Water</td>
                    <td>3.75 uL</td>
                    <td>9.5 uL</td>
                    <td>19 uL</td>
                    <td></td>
                  </tr>
                  <tr>
                    <td>Template DNA</td>
                    <td>1 uL</td>
                    <td>0.5 uL</td>
                    <td>1 uL</td>
                    <td><1,000 ng</td>
                  </tr>
                  </table>
                </li>
                <li>Mix throuroghly and spin down the PCR reactions.</li>
                <li>Place the tubes in the PCR Machine and use the following settings:
                  <ol>
                    <li>Initial denaturation - 98°C for 30 sec.. </li>
                    <li>Separation of strands - 98°C for 10 sec.</li>
                    <li>Annealing of primers – Tm of primer (approx. 55-62°C) for 30 sec.</li>
                    <li>Elongation – 72°C for 30 sec/kb. </li>
                    <li>Repeat step 1-4 for 30 cycles.</li>
                    <li>Final extension - 72°C for 10 min. </li>
                    <li>Hold - 4°C for infinite time. </li>
                  </ol>
                  <i>Note: If smearing appears on gel, it could be due to too many cycles – optimize between 25-35 cycles. If no correct bands appear on gel, try doing a temperature gradient PCR to optimixe the annealing temperature of the primers.</i>
                </li>
                <li>Once the PCR tubes are taken out of the PCR Machine, turn it off and leave the lid closed until it has reached room temperature, then open it and leave the lid open for the next user.</li>
              </ol>
              Source: <a href="https://www.neb.com/protocols/2012/08/29/protocol-for-q5-high-fidelity-2x-master-mix-m0492 ">here</a>
            </div>
          </div>
        </div>
      </div>

      <div class="well experiment-container">
        <a data-toggle="collapse" data-parent="#experiments" href="#cpcr">
          <h5>
            <i class="fa fa-flask" aria-hidden="true"></i>
            Colony PCR
           </h5>
        </a>
        <div id="cpcr" class="collapse">
          <div class="row">
            <div class="col-md-6">
              <u>Material</u>
              <ul>
                <li>70% ethanol</li>
                <li>Transformed cells on plate</li>
                <li>Milliq Water</li>
                <li>Q5 High-Fidelity 2X Master Mix</li>
                <li>10uM Forward Primer</li>
                <li>10uM Reverse Primer</li>
                <li>Template DNA</li>
                <li>1% Agarose Gel</li>
              </ul>
              <u>Equipment</u>
              <ul>
                <li>PCR Machine</li>
                <li>Pipettor</li>
                <li>Pipette tips</li>
                <li>Vortexer</li>
                <li>Table Top Centrifuge</li>
                <li>PCR Tubes</li>
                <li>Agarose Gel Equipment</li>
              </ul>
            </div>
            <div class="col-md-6">
              <u>Procedure</u>
              <ol>
                <li>Clean your working area by wiping down with 70% ethanol.</li>
                <li>Label one PCR tube for each colony, at least three colonies should be picked for screening per plate.</li>
                <li>Add 10uL of Milliq water to each labelled PCR tube.</li>
                <li>Pick the colonies carefully using a pipett tip and resuspend them into the prepared PCR tubes. </li>
                <li>Prepare a 12.5uL PCR reaction for each colony, where the addition of template DNA is taken from the PCR tubes with the resuspended colonies and 1uL should be added. </li>
                <li>Mix throuroghly and spin down the PCR reactions.</li>
                <li>Place the tubes in the PCR Machine and use the following settings:
                  <ol>
                    <li>Initial denaturation - 98°C for 10 min.</li>
                    <li>Separation of strands - 98°C for 10 sec.</li>
                    <li>Annealing of primers – Tm of primer (approx. 55-62°C) for 30 sec.</li>
                    <li>Elongation – 72°C for 30 sec/kb. </li>
                    <li>Repeat step 1-4 for 30 cycles.</li>
                    <li>Final extension - 72°C for 10 min. </li>
                    <li>Hold - 4°C for infinite time. </li>
                  </ol>
                  <i>Note: If smearing appears on gel, it could be due to too many cycles – optimize between 25-35 cycles. If no correct bands appear on gel, try doing a temperature gradient PCR to optimixe the annealing temperature of the primers. Also, cPCR is not enough accurate – so sending for sequencing could still be an idea even if no bands appear on the gel.</i>
                </li>
                <li>Once the PCR tubes are taken out of the PCR Machine, turn it off and leave the lid closed until it has reached room temperature, then open it and leave the lid open for the next user.</li>
                <li>Prepare a 1%-agarose gel with the number of wells corresponding to the number of colonies screened.</li>
              </ol>
              Source: <a href=" https://www.neb.com/protocols/2012/08/29/protocol-for-q5-high-fidelity-2x-master-mix-m0492">here</a>
            </div>
          </div>
        </div>
      </div>
      <div class="well experiment-container">
        <a data-toggle="collapse" data-parent="#experiments" href="#cpcr">
          <h5>
            <i class="fa fa-flask" aria-hidden="true"></i>
            Colony PCR
           </h5>
        </a>
        <div id="cpcr" class="collapse">
          <div class="row">
            <div class="col-md-6">
              <u>Material</u>
              <ul>
                <li>GeneJET Plasmid Miniprep Kit (ThermoScientific)</li>
                <li>96% ethanol</li>
                <li>Sterile MQ water</li>
                <li>Cell culture</li>
              </ul>
              <u>Equipment</u>
              <ul>
                <li>Centrifuge</li>
                <li>Vortex (optionally)</li>
                <li>Pipettor</li>
                <li>100 mL Pasteur pipettes</li>
                <li>Lab timer</li>
                <li>Pipettes and tips (100 &micro;)
              </ul>
            </div>
            <div class="col-md-6">
              <u>Procedure</u>
              <ol>
                <li>Add 250 mL dH2O to beaker</li>
                <li>Add all dry reagents</li>
                <li>Add another 250 mL dH2O, stir and autoclave</li>
              </ol>
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>
</div>

</html>
{{Lund/footer}}
