{{Lund}}
<html>

<script type="text/javascript">
  $('#navbar').addClass('navbar-transparent');
</script>

<header class="modeling-landing">
  <div class="container">
    <h1 class="display">Modeling</h1>
  </div>
</header>

<div class="container">
  <ul class="nav nav-pills nav-auto-stacked">
    <li role="presentation" class="active"><a href="/Team:Lund/Model.html">Results</a></li>
    <li role="presentation"><a href="/Team:Lund/Modeling/ModelDerivation.html">Model derivation</a></li>
    <li role="presentation"><a href="/Team:Lund/Modeling/ParameterEstimation.html">Parameter estimation</a></li>
    <li role="presentation"><a href="/Team:Lund/Modeling/Implementation.html">Implementation</a></li>
  </ul>
</div>

<section id="modeling">
  <div class="container">
    <h3>Overview</h3>

    <p>In this section presents the results of our model. In the model dervation section we present the derivation with the corresponding assumptions
    and equations. In the parameter estimation section we list the parameters used and how they were estimated. For a description of the
    implementation details, see the implementation section and for an overview of the biology, see the design section. </p>

    <h3>Purpose</h3>

    <p>An important part in engineering is the simulation of systems in order to yield an insight of the expected behaviours before actual
    implementations are carried out. This process can be both resource and time saving as practical implementations are often expensive and tedious to
    carry out. The same principle can be applied in synthetic biology where biological constructs are tested in silica prior to wet lab
    implementations. By simulating the behaviours, it is possible in an early stage of the development to evaluate whether the construct will suffice
    it's intended application. This is especially important for complex systems where basic heuristics no longer can give neither sufficient
    predictions or mechanistic understandings of the underlying components. </p>

    <p>The purpose of this model is to study the time dynamics of a <i>lac</i> regulated AND-gate based on a tripartite association of green
    fluorescent proteins induced by salicylate and estrogen. The intentions are to study the dose-response behaviours with respect to <i>two</i>
    inputs. Interesting parameters are the dose-response sensitivities and the dose-response times as these play an important part in real life
    applications of biosensors. Additionally, it is interesting to study the underlying biochemical mechanics in order to establish an understanding
    of the behaviours. Also, would a biosensor of this kind be sufficient for the detection of two concurrently present molecules?</p>


    <h3>Dose-response analysis</h3>

    <h5>Detection limit</h5>
    <p>In this section we study the biosensors dose-response characteristics. <i>fig. 1</i> illustrates the dose-response landscape for the system.
    The surface was generated by letting the system evolve until steady state (5000 min) without any inducers, followed by the simultaneous addition
    of IPTG, salicylate and estrogen. A concentration of 1 mM IPTG was added and the salicylate and estrogen were varied  in the range 0-1000 nM and
    0-1000 nM for 40 steps respectively. The system was then further simulated until equilibrium (5000 min) and the steady state concentrations
    of the associated GFP were saved. From the figure, it can be seen that for low concentrations of salicylate and estrogen, a low response
    is obtained. As the concentrations are increased, the corresponding response is increased until reaching a plateau were the response is saturated for
    increased concentrations of pollutants, giving the landscape the characteristic mountain look. This is reasonable as the amount of GFP expressed
    is limited by the amount of GFP1-9 and GFP10-ER-GFP11. For instance, adding more salicylate will eventually make all the <i>Psal</i>
    transcriptionally active and the addition of more will thus not increase the expression of GFP1-9. In a similar manner, if all GFP10-ER-GFP11 has
    been activated and associated with GFP1-9, the addition of estrogen will not cause more GFP10-ER-GFP11 to be expressed. Consequently, if either
    one of proteins are abscent, a complementation cannot occur and the response remains unchanged. </p>


    <figure style="text-align:center">
      <img src="../../assets/Modeling/T--Lund-dose_response_landscape_40x40.svg" alt="Reaction scheme">
      <figcaption>
        <i>Fig. 1: Dose-response landscape for the system. The data was generated by running simulations with salicylate and estrogen  concentrations in
        the range 0-1000 in steps of 40, yielding a total of 1600 points. The simulations were run using 1 mM IPTG. A characteristic mountain-like landscape
        can be observed where the response reaches a plateau for high concentrations of salicylate and estrogen.</i>
      </figcaption>
    </figure>


    <br>


    <p><i>Fig. 2</i> shows the dose-response heat map for the system obtained by looking at the surface level of <i>fig. 1</i> in the
    salicylate-estrogen plane. From the figure, the sensitivity with respect to salicylate and estrogen can be studied. It can be seen that no respose
    is generated for low concentrations of both inputs. In addition, in the abscence of either molecule, no output is generated regardless if the
    other exists in excess or not. This clearly illustrates the AND-behaviour and it can be seen that the detection limits are approximately 1 nM and
    100 nM for salicylate and estrogen respectively. Concentrations lower than those yield either no or a weak response relative to the maximal
    response. Clearly, this indicates a very sensitive system with respect to the inputs. Since the detection limit of extracellular salicylate is 1
    nM and since the model assumes free diffusion, this would correspond to roughly 1 nM intracellular salicylate which is almost equal to one
    molecule. This means that it would be possible yield a response based of one single molecule entering the cell. This seem to be an overestimate of
    sensitivity since we have not found any literature on whole cell biosensors where subnanomolar detection limits have been attained. Nevertheless,
    the results indicate that the sensor could be highly sensitive in the codetection of salicylate and estrogen. </p>


    <p>Moreover, since the model manages to measure quantities in the order of single molecules, one might question the validity on using a continuous
    scheme. For example, there is a very weak response obtained for a salicylate concentration of 0.5 nM, corresponding to one half molecule. It might
    thus seem more reasonable to model the biosensor using discrete and stochastic methods in the single cell level, e.g. via the Gillespie algorithm. However,
    since we are considering a deterministic scheme interpreted as the average response over a large population, we make no assumption as to the
    homogenity of the molecular distribution among the cells. That is, even though the concentration of a molecule in one cell corresponds to one half
    molecule, it does not mean that the cell has a half molecule in the cytoplasm, but rather that the average amount of molecules over the population
    is one half salicylate. This means that we are studying the deterministic limit for one cell and the results should be interpreted as an expected
    average of a large population. We thus consider the choice to model the cell deterministic reasonable as we do no expect a biosensor to consist of
    one single cell. </p>


    <figure style="text-align:center">
      <img src="../../assets/Modeling/T--Lund-dose_response_heatmap_40x40.svg" alt="Reaction scheme">
      <figcaption>
        <i>Fig 2: Dose-response heatmap for the system. The data was generated by running simulations with salicylate and estrogen concentrations in the
        range 0-1000 in steps of 40, yielding a total of 1600 points. The simulations were run using 1 mM IPTG. The yellow region in the top left corner
        illustrates the detection region where the system is able to respond on salicylate and estrogen.</i>
      </figcaption>
    </figure>

    <br>


    <h5><i>lac</i> regulation </h5>
    <p> With the use of <i>lac</i> inducible promoters, it is possible to have an on/off switch regulating the AND-gate. The dose-response landscape
    and heat map obtained without any IPTG inducer but otherwise same conditions as in <i>fig. 1</i> and <i>fig. 2</i> is shown in <i>fig. 3</i>.
    A slight leak response can be observed which is because the <i>lac</i> repression does not inactivate the expression entirely. Since the repressing
    dynamics is modeled as a sequence of reversible reactions, there will always be a small amount of unbound operators allowing for transcription to
    occur. Nevertheless, when comparing the expression levels with the induced case in <i>fig. 1</i> we see that the difference is at least four
    orders of magnitude, clearly indicating a working switch. </p>


    <figure style="text-align:center">
      <img src="../../assets/Modeling/T--Lund-landscape_noIPTG.svg" alt="Reaction scheme">
      <figcaption>
        <i>Fig. 3: Dose-response landscape (upper) and heatmap (lower). The data was generated by running simulations with salicylate and estrogen
        concentrations in the range 0-1000 in steps of 40, yielding a total of 1600 points. The simulations were run using no IPTG. It can be seen that
        although there exist a slight leakage, the response is still at least four orders magnitude lower than in the IPTG induced case. </i>
      </figcaption>
    </figure>

    <br>

    <h3>Time characteristics</h3>


    <h5>Sample simulations</h5>

    <p> <i>Fig. 4</i> illustrates four simulations showing the time evolution of the associated GFP, GFP1-9 and estrogen bound GFP10-ER-GFP11. The
    simulations were run as usual (see implementation section for details), using salicylate and estrogen concentrations in the range 1-100 nM and
    10-300 nM respectively. It can be seen that the time evolutions are smooth as the species converges to steady state concentrations and that no
    oscillations are present. There are thus no observed undershoots or overshoots as a response to salicylate and estrogen. The rise time for the
    associated GFP, defined as the time it takes to go from 10% to 90% of the steady state value was measured at approximately 650 min for all
    realisations. The settling time, defined as the time it takes to fall within 2% of the steady state concentration was measured at roughly 1200 min
    for all simulations. </p>

    <figure style="text-align:center"> <img src="../../assets/Modeling/T--Lund-many_timeplots3.svg" alt="Reaction scheme"> <figcaption> <i>Fig. 4:
    Four simulations showing the time evolution of the associated GFP (GFP), GFP1-9 (GFP1-9) and estrogen bound GFP10-ER-GFP11 (GFP10-ER-GFP11). The
    simulations were carried out using the setup explained in the implemenetation section.</i> </figcaption> </figure>

    <h5>Gene regulation</h5>

    <p> The dynamics of the <i>lac</i> operators prior to the addition of IPTG, salicylate and estrogen is shown in <i>fig. 5</i>. The upper part of
    the figure shows the repression of the <i>lac</i> operators regulating the expression of NahR and GFP10-ER-GFP11. It can be seen that all
    operators are repressed within a couple of minutes. However, within this timeframe both proteins are transcribed until completely repressed. This
    is noted in the lower part of the figure where expressed NahR binds to the <i>Psal</i> promoter, inverting the ratio between the protein bound
    promoter <i>NPsal</i> and <i>Psal</i>. However, as the operators are repressed, the expression of NahR alters and the equilibrium is displaced
    back to <i>Psal</i> as the NahR is decomposed. Worth noting is that the entire process takes almost 1000 min before reaching equilibrium. This can
    be traced to the half-life of NahR which was set to 2 hours and the slow <i>Psal</i> dynamics. </p>

    <figure style="text-align:center"> <img src="../../assets/Modeling/T--Lund-operators_NOindudction_100x300.svg" alt="Reaction scheme"> <figcaption>
    <i>Fig. 5: Illustration of the gene regulating dynamics prior to the addition of inducers. The simulation was generated using the setup explained
    in the implementation section. In the upper part the <i>lac</i> dynamics are shown and it can be seen that all operators are repressed at
    equilibrium. In the lower part the <i>Psal</i> dynamics are shown. It can be seen that the equilibrium is established first after 1000 min once
    the transients has died. </i> </figcaption> </figure>
    <br>

    <p>The dynamics of the <i>lac</i> operators and the <i>Psal</i> promoter after the addition of IPTG, salicylate and estrogen is shown in <i>fig.
    6</i>. Here, 100 nM and 300 nM of salicylate and estrogen was used and the IPTG concentration was set at 1 mM. In the upper part of the figure, it
    can be seen that all operators are repressed at time zero, completely inhibiting the transcription of NahR and GP10-ER-GFP11. After the addition
    of IPTG, the operators are derepressed which consquently increases the transcription rates. It can be seen that not all repressors are inhibited
    at steady state, indicating that higher expression levels can be achieved by the addition of more IPTG. However, we chose to limit the
    extracellular concentration to 1 mM to keep the simulation within realistic boundaries set by wet-lab working conditions [1]. In addition, it is
    also worth noting that we did have some issues with numerical instability for higher concentrations of IPTG. This can be traced to the shock
    effect which occurs when a high concentration of IPTG is added momentaneously at time zero. However, it was observed that this problem could be
    circumvented by gradually increasing the concentration over a short time span instead of adding everything simultaneously.</p>


    <p> The dynamics of the <i>Psal</i> promoter is illustrated in the lower part of <i>fig. 6</i>. At time zero, only <i>Psal</i> is present due to the
    abscence of NahR and salicylate. As time progresses, NahR is expressed, forming the intermediate NahR bound promoter denoted as NPsal. This is
    followed by salicylate simultaneously diffusing over the cell membrane and binding to NPsal. What is observed is then the <i>Psal</i> promoter
    slowly decreasing in concentration while <i>NPsal</i> is formed, followed by a lagged increase of the active promoter <i>SNPsal</i>. It can be
    seen that all <i>Psal</i> is converted into the active form <i>SNPsal</i> within the range of approximately 200 min. This indicates that the
    expression of GFP1-9 does not reach the maximal rate until 200 min after induction, which corresponds to almost a third of the total rise time.</p>


    <figure style="text-align:center"> <img src="../../assets/Modeling/T--Lund-operators_indudction_100x300.svg" alt="Reaction scheme"> <figcaption>
    <i>Fig. 6: Illustration of the gene regulating dynamics after the addition of inducers. The simulation was generated using the setup explained in
    the implementation section with an IPTG concentration of 1 mM and salicylate and estrogen concentration of 100 nM and 300 nM respectively. The
    upper part shows the <i>lac</i> dynamics and it can be seen that not all operons are free after the addition of IPTG. In the lower part the
    <i>Psal</i> dynamics are shown. It can be seen that an equilibrium is established after approximately 200 min where equilibrium is shifted towards
    <i>SNPsal</i>. </i> </figcaption> </figure>



    <h5>Response time analysis</h5>

    <p> From the simulations carried out in the preceeding section, it was noted that rise times in the order of 650 min were observed for a variety
    of doses and it was seen that the time varied little with respect to the applied doses. Due to the long rise time of almost 11 hours, it would be
    of great interest to reduce the time to allow for a rapid detection during practical uses. Hence, to investigate the role of the parameters we
    choose to do global sensitivity analysis with respect to the rise time. Global sensitivity analysis is a method used to study the effect that
    large variations in the input parameters have on the output. This is slightly different from local sensitivity analysis which studies the effect
    on the output caused by small pertubtations on the input. One can view the local variant as a gradient of the output with respect to the parameter
    space at the point in the parameter space considered, whereas the global variant gives an overall effect on the output with respect to a specified
    domain in the parameter space [2].</p>

    <p> The parameters for the sensitivity analysis were chosen with respect to their expected effect on the rise time as well as their capability of
    being modified in the lab and are illustrated in <i>table 1</i>. We choose to investigate the transcription and translation rates for NahR, GFP1-9
    and GFP10-ER-GFP11, the <i>Psal</i> dynamics, the ligand binding dynamics of estrogen to GFP10-ER-GFP11 and the tripartite GFP association rate.
    The parameter space was chosen by logarithmic sampling around the estimated value within one orders of magnitude, that is the interval with bounds
    specified by ten times and one tenth times the parameter. These intervals were chosen somewhat arbitrarly based on the belief that it is not
    possible (or at least very hard) to alter the parameters more than ten times their original values. We think it's worth mentioning that we
    initially had issues when sampling over the normal parameter space as it resulted in very small sensitivities for all parameters. This was solved
    by sampling over the corresponding logarithmic parameter space. In total, 84 000 samples were generated using Saltelli's sampling scheme (see
    implementation) and the sensitivities can be seen in <i>table 1</i>. Here, S<sub>1</sub> denotes om the first order sensitivity index which is a
    measure of the contribute to the variance in the output caused the parameter alone.

    </p>

    <table class="table">
      <caption>Table 1: Parameters chosen for the global sensitivity analysis</caption>
      <thead>
        <tr>
          <th>Parameter</th>
          <th>Value</th>
          <th>Scanned range</th>
          <th>S<sub>1</sub></th>
          <th>S<sub>2</sub></th>
          <th>S<sub>3</sub></th>
          <th>95% CI</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>$\ce{k_\text{aGFP}}$</td>
          <td>$1.8 \cdot 10^{-5}$</td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
        </tr>
        <tr>
          <td>$\ce{k_\text{aEEs}}$</td>
          <td>$0.0078$</td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
        </tr>
        <tr>
          <td>$\ce{k_\text{dEEs}}$</td>
          <td>$0.072$</td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
        </tr>

        <tr>
          <td>$\ce{k_\text{aNP}}$</td>
          <td>$4.7 \cdot 10^{-3}$</td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
        </tr>
        <tr>
          <td>$\ce{k_\text{dNP}}$</td>
          <td>$11.5$</td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
        </tr>
        <tr>
          <td>$\ce{k_\text{aSNP}}$</td>
          <td>$4.61 \cdot 10^{-4}$</td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
        </tr>
        <tr>
          <td>$\ce{k_\text{dSNP}}$</td>
          <td>$3.81 \cdot 10^{-6}$</td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
        </tr>
        <tr>
          <td>$\ce{k_\text{MN}}$</td>
          <td>$3.09$</td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
        </tr>
        <tr>
          <td>$\ce{k_\text{ME}}$</td>
          <td>$3.18$</td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
        </tr>
        <tr>
          <td>$\ce{k_\text{MG}}$</td>
          <td>$0.168$</td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
        </tr>
        <tr>
          <td>$\ce{k_\text{N}}$</td>
          <td>$3.0$</td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
        </tr>
        <tr>
          <td>$\ce{k_\text{E}}$</td>
          <td>$2.86$</td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
        </tr>
        <tr>
          <td>$\ce{k_\text{G}}$</td>
          <td>$4.59$</td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
          <td></td>
        </tr>
      </tbody>
    </table>

    <p>Clearly this indicates a highly insensitive system with respect to the parameters. Although this is not uncommon in biological systems [länk],
    we suggest the experiments to be repeated by someone else to ensure this is the actual case. </p>

    <h5>Parameter sweeps</h5>


    <p>To illustrate the sensitivities we perform parameter scans and plot the parameter values with respect to the response times. a</p>

    <p>Gör några parameter-sweeps där du visar hur okänslig systemet är. Gör några map k_aGFP och andra så att hastigheterna blir instantiva.</p>
    <p> <b>Global känslighetsanalys där vi kollar på vad som påverkar responstiden samt hur man skulle kunna minska den</b> <br> <b>1. Kolla
    känslighetsanalysen på vilka parametrar som påverkar responstiden signifikant. <br> 2. Kör parameter sweeps of visa effekten </b> <br> When using
    the construct for practical applications, it would be of interest to reduce the rise time to allow for a rapid detection of the solutes. Figure ...
    illustrates a global sensitivity analysis carried out on

    <br><br>
    The long response times can be traced back to the tripartite split GFP association rate constant which was estimated at a value of 1.8e-5 nm/min.
    Clearly, this makes the biosensor inherently slow unless optimizations at the primary structure of the protein is made to facilitate the
    association. As such, as indicated by the sensitivity analysis in figure …. Changing the promoter strength … does not improve the response times
    significantly. </p>


    <h3>Conclusion</h3>
    <p>The model suggests that ... återkoppla till purpose och hur detta har påverkat vår design och implementering.  </p>

    <h3>References</h3>
    <p>[1] Larentis, A., Nicolau, J., Esteves, G., Vareschini, D., de Almeida, F., dos Reis, M., Galler, R. and Medeiros, M. (2014). Evaluation of pre-induction temperature, cell growth at induction and IPTG concentration on the expression of a leptospiral protein in E. coli using shaking flasks and microbioreactor. BMC Research Notes, 7(1), p.671.</p>
    <p>[2] Zi, Z. (2011). Sensitivity analysis approaches applied to systems biology models. IET Systems Biology, 5(6), pp.336-346.</p>


  </div>
</section>
</html>
{{Lund/footer}}
