{{Lund}}
<html>
</div> <!-- iGEM stuff-->

<script type="text/javascript">
  $('#navbar').addClass('navbar-transparent');
</script>

<header class="modeling-landing">
  <div class="container">
    <h1 class="display">Modeling</h1>
  </div>
</header>

<section id="modeling">
  <div class="container">
  <nav class="navbar navbar-default navbar-static-top nav nav-pills" style="background-color:white">
    <li><a href="/Team:Lund/Model.html">Results</a></li>
    <li class="active"><a href="/Team:Lund/Modeling/ModelDerivation.html">Model derivation</a></li>
    <li><a href="/Team:Lund/Modeling/ParameterEstimation.html">Parameter estimation</a></li>
    <li><a href="/Team:Lund/Modeling/Implementation.html">Implementation</a></li>
  </nav>
  <h3>Overview</h3>
  <p>Om modellen: vi modellerar pETDuet-1 med två lac operons, en gen som kodar repressorn samt två vägar som genererar gfp1-9 och er-alphalbd för slutgiltig komplementation. MOdellen för lac operonet har vi tagit från en artikel och vi presenterar endast modellen kort. För mer info hänvisar vi därför läsaren till originalartikeln. Resten bygger på proportionalitetsantaganden baserat på kontemporär litteratur.</p>
  <p><b>[Bild på systemet]</b> </p>
  <h3>Modeling lac operon</h3>
  <p>asjkdhaskd</p>
  <h3>Modeling NahR</h3>
  <p>It has been shown that NahR binds to the Psal promoter as a transcription factor regardless of the presence of salicylate [1989]. The binding affinity has been thought to be unaffected by the salicylate [1989], but recent in-vitro experiments with surface plasmon resonance spectroscopy indicate that salicylate actually has an inhibiting effect on the binding to Psal [SPR]. There has also been indications that NahR binds as a tetramer to the promoter [1990], but recent findings has shown that it binds sequentially the promoter to then undergo miltimerization on site [SPR]. The degree of multimerization is to our knowledge not known and there are indications that the pSal promoter can have up to eight bound monomers [SPR]. </p>
  <p>
    Regardless, as a first order approximation (and due to the numerical ill-posedness that comes with higher order systems) we choose to model both the binding of the NahR to pSal as a first order equilibrium sa well as the subsequent binding of salicylate to the NahR-Psal complex. The resulting Salicylate-Nahr-Psal complex then starts transcibing the gfp1-9 transcript. The wholer esults in a Michaelis-Menten like system:
    $$\ce{N + P <--> NP}$$
    $$\ce{NP + S <--> SNP}$$
    $$\ce{SNP -> SNP + M_G}$$
  </p>
  <p>The following reactions .... </p>

  <h3>Modeling hER</h3>
  <p>
    The human estrogen receptor ER-&alpha; is modeled using simple reaction schemes. The binding of the estrogen antagonist $\ce{E}$ onto the estrogen receptor alpha ligand binding domain $\ce{ER}$ is modeled as a second order reaction where the active estrogen receptor state $\ce{\text{a}ER}$ is formed. The active state can decompose into $\ce{E}$ and $\ce{ER}$, but since the equilibrium constant is strongly shifted towards the binding formation (see parameter estimation), we model it as an irreversible reaction.
    $$\ce{ER + E ->[k_{ae}] \text{a}ER}$$
  </p>
  <p>
    The entry of the antagonist $\ce{E}$ is modeled by assuming an extracellular unit $\ce{E_\text{ex}}$ which diffuses over the cell membrane.
    $$\ce{E_\text{ex} <-->[k] E}$$
    It is assumed later that the amount of $\ce{E_\text{ex}}$ is constant and is thus set as a boundary condition.
  </p>
  <p>
    The degradations are all of first order.
    $$\ce{ER -> \phi}$$
    $$\ce{E -> \phi}$$
  </p>
  <h3>Modeling the complementation</h3>
  <p>
    djfgdI  g
  </p>
  <h3>Determinstic equations</h3>
  <p>ajsfasf</p>

  <div class="column half_size">
      <h3>Model derivation</h3>
      <p>
        This section presents the derivation of the model. We use a simplified model
        of the <i>lac</i> operon where we dont take into account of the expression of
        <i>lac</i> repressor or cytoplasmic free repressors (mer...) This was done to keep
        the model simple due to the lack of experimental data. It that the amount of data needed
        increases exponentially with the amount of parameters used in a system, also called
        curse of dimensionality ... yada yada yada.
      </p>

      <p>
        A mechanistic scheme of the model is illustrated in figure x (gör chemdrawbild) and a list
        of the model species is shown in table 1. Initially,
        it is assumed that the two operons, $\ce{O_\text{N}}$ and $\ce{O_\text{E}}$, which control the
        expression of NahR and ER-$\alpha$ exist exclusively in their repressed forms, $\ce{RO_\text{N}}$
        and $\ce{RO_\text{E}}$, where $\ce{R}$ is the <i>lac</i> repressor. Through the addition of an inducer,
        $\ce{I}$, the operons are induced to allow for expression. It has been shown that two inducers
        are required to induce an peron [19 i lac].

        $$\ce{2 I + RO_\text{N} <-->[k_1][k_2] RI_2 + O_\text{N}}\tag{1}$$
        $$\ce{2 I + RO_\text{E} <-->[k_1][k_2] RI_2 + O_\text{E}}\tag{2}$$

        Thereafter, $\ce{O_\text{N}}$ and $\ce{O_\text{E}}$ are transcribed to yield $\ce{M_\text{N}}$ and $\ce{M_\text{E}}$ which
        are translated into NahR, $\ce{N}$ and the estrogen receptor alpha ligand binding domain, $\ce{E}$.
        Both reactions are assumed to be self catalyzed and first order.

        $$\ce{O_\text{N} ->[k] O_\text{N} + M_\text{N}}\tag{3}$$
        $$\ce{O_\text{E} ->[k] O_\text{E} + M_\text{E}}\tag{4}$$

        $$\ce{M_\text{N} ->[k] M_\text{N} + N}\tag{5}$$
        $$\ce{M_\text{E} ->[k] M_\text{E} + E}\tag{6}$$

        It has been shown that the ligand binding of estrogen analogs, $\ce{Es}$, to $\ce{E}$ are of first order
        and that the consequent active ligand binding receptor complex, $\ce{aE}$, self dissociates in a first
        order reaction [kinetic analysis er$\alpha$]. We assume that the linkers and the gfp fragments coupled
        to the ligand binding domain do not alter the association kinetics.

        $$\ce{E + Es <-->[k_\text{a}][k_\text{d}] \text{a}E}\tag{7}$$

        It has been shown that NahR binds to the Psal promoter as a transcription factor regardless of the presence
        of salicylate [1989]. The binding affinity has previously been thought to base unaffected
        of salicylate, but recent in vitro studies with surface plasmon resonance indicate that
        it in fact has an inhibinting impact on the binding to Psal [SPR]. There has also been indications
        that NahR binds as a tetramer to the promoter [1990]. However, recent studies has shown
        that NahR actually binds sequentially as a monomer to the promoter and then undergoes a multimerization
        on site [SPR]. The degree of multimerization is to our knowledge ambiguous as immobilized
        Psal has had up to eight bound monomers. In addition, we have not been able to find the salicylate
        binding mechanisms to the Psal-NahR complex. We do thus not know wether one salicylate has to binds to
        each NahR before transcription can occur or wether the activity is increased gradually
        with each intermediately bound solute. Neither do we know how the transcriptional activity
        changes with each intermediate NahR monomer bound to Psal. Assuming all intermediate steps exist
        and contribute to the transcription, the model would quickly increase in size and complexity, yielding a
        big parameter space. To be able to model this correctly, there is a need of a big amount of data
        as the amount of data increases exponentially with each extra parameter.
        As we only have a sparse amount of data, we choose to model the pathway in two steps, where the NahR, $\ce{N}$,
        binds to Psal in a second order reaction. Consequently, one salicylate binds to the complex to
        allow for transcription.

        <b>Testa skriva något i stil med "One could image modeling the pathway as a series of reactions where with each extra nahr bound monomer a new equilibrium can occur where several salicylates bind, yielding different degrees of transcriptions."</b>I

        $$\ce{N + P\text{sal} <-->[k_1][k_2] NP\text{sal}}\tag{8}$$
        $$\ce{S + NPsal <-->[k_1][k_2] SNPsal}\tag{9}$$
        $$\ce{SNPsal ->[k] SNPsal + M_\text{G}}\tag{10}$$
        $$\ce{M_\text{G} ->[k] M_\text{G} + G}\tag{11}$$


        </p>

      <table class="table">
        <caption>Table 1: List of species</caption>
        <thead>
          <tr>
            <th>Notation</th>
            <th>Description</th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>$\ce{O_\text{R}}$</td>
            <td><i>lacI</i> gene</td>
          </tr>
          <tr>
            <td>$\ce{M_\text{R}}$</td>
            <td>LacI mRNA</td>
          </tr>
          <tr>
            <td>$\ce{R}$</td>
            <td>LacI monomer</td>
          </tr>
          <tr>
            <td>$\ce{R_2}$</td>
            <td>LacI Repressor</td>
          </tr>
          <tr>
            <td>$\ce{O_\text{A}}$</td>
            <td><i>lac</i> operator for protein $\ce{A}$</td>
          </tr>
          <tr>
            <td>$\ce{O_\text{B}}$</td>
            <td><i>lac</i> operator for protein $\ce{B}$</td>
          </tr>
          <tr>
            <td>$\ce{R_2O_\text{A}}$</td>
            <td>Repressed <i>lac</i> operator for protein $\ce{A}$</td>
          </tr>
          <tr>
            <td>$\ce{R_2O_\text{B}}$</td>
            <td>Repressed <i>lac</i> operator for protein $\ce{B}$</td>
          </tr>
          <tr>
            <td>$\ce{I_\text{ex}}$</td>
            <td>Extracellular inducer</td>
          </tr>
          <tr>
            <td>$\ce{I}$</td>
            <td>Intracellular inducer</td>
          </tr>
          <tr>
            <td>$\ce{I_2R_2}$</td>
            <td>Inhibited $\ce{LacI}$ repressor</td>
          </tr>
          <tr>
            <td>$\ce{M_\text{A}}$</td>
            <td>mRNA protein $\ce{A}$</td>
          </tr>
          <tr>
            <td>$\ce{M_\text{B}}$</td>
            <td>mRNA protein $\ce{B}$</td>
          </tr>
          <tr>
            <td>$\ce{A}$</td>
            <td>Protein $\ce{A}$</td>
          </tr>
          <tr>
            <td>$\ce{B}$</td>
            <td>Protein $\ce{B}$</td>
          </tr>
        </tbody>
      </table>

    </div>

    <div class="clear"></div>

    <div class="column half_size">
      <h3>Determinstic equations</h3>
    </div>

    <div class="clear"></div>


  </div>
</div>
</section>
</html>
{{Lund/footer}}
