{{Lund}}
<html>
</div> <!-- iGEM stuff-->

<script type="text/javascript">
  $('#navbar').addClass('navbar-transparent');
</script>

<header class="modeling-landing">
  <div class="container">
    <h1 class="display">Modeling</h1>
  </div>
</header>

<section id="modeling">
  <div class="container">
  <nav class="navbar navbar-default navbar-static-top nav nav-pills" style="background-color:white">
    <li><a href="/Team:Lund/Model.html">Results</a></li>
    <li><a href="/Team:Lund/Modeling/ModelDerivation.html">Model derivation</a></li>
    <li class="active"><a href="/Team:Lund/Modeling/ParameterEstimation.html">Parameter estimation</a></li>
    <li><a href="/Team:Lund/Modeling/Implementation.html">Implementation</a></li>
  </nav>
  <h3>Overview</h3>
  <p>In this section we provide the parameters used in the model. Table 1 yields an overview for all parameters with a
  corresponding reference to the literature and comment in the description section.</p>
  <br>

  <h3>Parameters</h3>
  <table class="table">
    <caption>Table 1: List of parameters.</caption>
    <thead>
      <tr>
        <th>Parameter</th>
        <th>Value</th>
        <th>Unit</th>
        <th>Description</th>
        <th>Reference</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>$\ce{C_\text{p}}$</td>
        <td>$40$</td>
        <td>-</td>
        <td>Copy number</td>
        <td>[1]</td>
      </tr>
      <tr>
        <td>$\ce{V}$</td>
        <td>$8 \cdot 10^{-16}$</td>
        <td>L</td>
        <td>E. coli cell volume</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{MR}}$</td>
        <td>$0.11$</td>
        <td>$\text{min}^{-1}$</td>
        <td>MR transcription rate constant</td>
        <td>[2]<sup>a)</sup></td>
      <tr>
      <tr>
        <td>$\ce{k_\text{R}}$</td>
        <td>$15$</td>
        <td>$\ce{min}^{-1}$</td>
        <td>R translation rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{2R}}$</td>
        <td>$50$</td>
        <td>$\ce{\text{nM}^{-1} min^{-1}}$</td>
        <td>R dimerization rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{-2R}}$</td>
        <td>$10^{-3}$</td>
        <td>$\ce{min}^{-1}$</td>
        <td>R$_2$ dissociation rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{r}}$</td>
        <td>$960$</td>
        <td>$\text{nM}^{-1} \text{min}^{-1}$</td>
        <td>Operator repression rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{-r}}$</td>
        <td>$2.4$</td>
        <td>$\text{min}^{-1}$</td>
        <td>Operator derepression rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{dr1}}$</td>
        <td>$3\cdot 10^{-7}$</td>
        <td>$\text{nM}^{-2}\text{min}^{-1}$</td>
        <td>Free R$_2$ repression rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{-dr1}}$</td>
        <td>$12$</td>
        <td>$\text{min}^{-1}$</td>
        <td>Free R$_2$ derepression rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{dr2}}$</td>
        <td>$3\cdot 10^{-7}$</td>
        <td>$\text{nM}^{-2} \text{min}^{-1}$</td>
        <td>Operator bound R$_2$ repression rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{-dr2}}$</td>
        <td>$4.8 \cdot 10^{3}$</td>
        <td>$\text{nM}^{-1} \text{min}^{-1}$</td>
        <td>Operator bound R$_2$ derepression rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{MN}}$</td>
        <td>$3.09$</td>
        <td>$\text{min}^{-1}$</td>
        <td>NahR mRNA transcription rate constant</td>
        <td>[3]<sup>b)</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{ME}}$</td>
        <td>$3.18$</td>
        <td>$\text{min}^{-1}$</td>
        <td>Estrogen receptor LBD mRNA transcription rate constant</td>
        <td>[3]<sup>b)</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{MG}}$</td>
        <td>$4.72$</td>
        <td>$\text{min}^{-1}$</td>
        <td>GFP1-9 mRNA transcription rate constant</td>
        <td>[3]<sup>b)</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{N}}$</td>
        <td>$3.0$</td>
        <td>$\text{min}^{-1}$</td>
        <td>NahR translation rate constant</td>
        <td>[3]<sup>c)</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{E}}$</td>
        <td>$2.86$</td>
        <td>$\text{min}^{-1}$</td>
        <td>Estrogen receptor LBD translation rate constant</td>
        <td>[3]<sup>c)</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{G}}$</td>
        <td>$4.59$</td>
        <td>$\text{min}^{-1}$</td>
        <td>GFP1-9 translation rate constant</td>
        <td>[3]<sup>c)</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{mG}}$</td>
        <td>$0.167$</td>
        <td>$\text{min}^{-1}$</td>
        <td>GFP1-9 maturation rate constant</td>
        <td>[3]<sup>d)</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{aEEs}}$</td>
        <td>$0.078$</td>
        <td>$\text{nM}^{-1}\text{min}^{-1}$</td>
        <td>Estrogen receptor-ligand association rate constant</td>
        <td>[4]<sup>e)</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{dEEs}}$</td>
        <td>$0.072$</td>
        <td>$\text{min}^{-1}$</td>
        <td>Estrogen receptor-ligand dissociation rate constant</td>
        <td>[4]<sup>e)</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{aNP}}$</td>
        <td>$4.7\cdot 10^{-3}$</td>
        <td>$\text{nM}^{-1}\text{min}^{-1}$</td>
        <td>NahR-Psal association rate constant</td>
        <td>[5]<sup>f)</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{dNP}}$</td>
        <td>$11.46$</td>
        <td>$\text{min}^{-1}$</td>
        <td>NahR-Psal dissociation rate constant</td>
        <td>[5]<sup>f)</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{aSNP}}$</td>
        <td>$3.16 \cdot 10^{-4}$</td>
        <td>$\text{nM}^{-1}\text{min}^{-1}$</td>
        <td>Salicylate-[NahR-Psal] association rate constant</td>
        <td>Estimated<sup>g)</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{dSNP}}$</td>
        <td>$14.4 \cdot 10^{-3}$</td>
        <td>$\text{min}^{-1}$</td>
        <td>Salicylate-[NahR-Psal] dissociation rate constant</td>
        <td>Estimated<sup>g)</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{aGFP}}$</td>
        <td>$1.8 \cdot 10^{-5}$</td>
        <td>$\ce{\text{n}M^{-1}min^{-1}}$</td>
        <td>Tripartite GFP association rate constant</td>
        <td>[6]<sup>h)</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{dGFP}}$</td>
        <td>$0.0$</td>
        <td>$\ce{min^{-1}}$</td>
        <td>Tripartite GFP dissociation rate constant</td>
        <td>[6]<sup>h)</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{dI}}$</td>
        <td>$0.92$</td>
        <td>$\text{min}^{-1}$</td>
        <td>IPTG diffusion constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{dS}}$</td>
        <td>$0.281$</td>
        <td>$\text{min}^{-1}$</td>
        <td>Salicylate diffusion rate constant</td>
        <td>[7]<sup>i)</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{dE}}$</td>
        <td>$1.44$</td>
        <td>$\text{min}^{-1}$</td>
        <td>Estrogen diffusion rate constant</td>
        <td>[8]<sup>j)</sup></td>
      </tr>
      <tr>
        <td>$\ce{\lambda_\text{MR}}$</td>
        <td>$0.462$</td>
        <td>$\ce{min^{-1}}$</td>
        <td>MR degradation rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{\lambda_\text{MN}}$</td>
        <td>$0.2$</td>
        <td>$\text{min}^{-1}$</td>
        <td>NahR mRNA degradation rate constant</td>
        <td>[3]<sup>k)</sup></td>
      </tr>
      <tr>
        <td>$\ce{\lambda_\text{ME}}$</td>
        <td>$0.2$</td>
        <td>$\text{min}^{-1}$</td>
        <td>Estrogen receptor LBD mRNA degradation rate constant</td>
        <td>[3]<sup>k)</sup></td>
      </tr>
      <tr>
        <td>$\ce{\lambda_\text{MG}}$</td>
        <td>$0.2$</td>
        <td>$\text{min}^{-1}$</td>
        <td>GFP1-9 mRNA degradation rate constant</td>
        <td>[3]<sup>k)</sup></td>
      </tr>
      <tr>
        <td>$\ce{\lambda_\text{R}}$</td>
        <td>$0.2$</td>
        <td>$\ce{min}^{-1}$</td>
        <td>R degradation rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{\lambda_{\text{R}_2}}$</td>
        <td>$0.2$</td>
        <td>$\ce{min}^{-1}$</td>
        <td>R$_2$ degradation rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{\lambda_{\text{I}_2\text{R}_2}}$</td>
        <td>$0.2$</td>
        <td>$\ce{min}^{-1}$</td>
        <td>I$_2$R$_2$ degradation rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{\lambda_\text{N}}$</td>
        <td>$0.0056$</td>
        <td>$\text{min}^{-1}$</td>
        <td>NahR degradation rate constant</td>
        <td>Assumed<sup>l)</sup></td>
      </tr>
      <tr>
        <td>$\ce{\lambda_\text{E}}$</td>
        <td>$0.0056$</td>
        <td>$\text{min}^{-1}$</td>
        <td>Estrogen receptor LBD degradation rate constant</td>
        <td>[9]<sup>m)</sup></td>
      </tr>
      <tr>
        <td>$\ce{\lambda_\text{EEs}}$</td>
        <td>$0.321$</td>
        <td>$\text{min}^{-1}$</td>
        <td>EE sdegradation rate constant</td>
        <td>[9]<sup>m)</sup></td>
      </tr>
      <tr>
        <td>$\ce{\lambda_\text{G}}$</td>
        <td>$0.321$</td>
        <td>$\text{min}^{-1}$</td>
        <td>GFP1-9 degradation rate constant</td>
        <td>Estimated<sup>g)</sup></td>
      </tr>
      <tr>
        <td>$\ce{\lambda_\text{GFP}}$</td>
        <td>$0.321$</td>
        <td>$\text{min}^{-1}$</td>
        <td>GFP degradation rate constant</td>
        <td>Estimated<sup>g)</sup></td>
      </tr>
    </tbody>
  </table>
  <br>
  <h3>Description</h3>
  <p>a)  The rate constant is 0.23 nM/min for an operator content of 2.08 nM [2]. This yields the rate constant $\ce{k_\text{MR} = 0.11}$ min<sup>-1</sup></p>

  <p>b)  The transcription rate is on average 50 nucleotides per second in <i>E. coli</i> [7]. The NahR transcript has approximately 900 nucleotides which
    yields $\ce{k_\text{MN}=3.33}$ min<sup>-1</sup>. The GFP10-linker-ER-&alpha;LBD-linker-GFP11 transcript has approximately 1800 nucleotides which
    yields $\ce{k_\text{ME}=1.67}$ min<sup>-1</sup>. The GFP1-9 transcript has approximately 600 nucleotides which yields $\ce{k_\text{MG} = 5}$
    min<sup>-1</sup>. </p>

  <p>c)  The translation rate is on average 15 amino acids per second in <i>E. coli</i> [7]. NahR has approximately 300 amino acids which
    yields $\ce{k_\text{MN}=3}$ min<sup>-1</sup>. GFP10-ER-&alpha;-GFP11 has approximately 600 amino acids which
    yields $\ce{k_\text{ME}=1.5}$ min<sup>-1</sup>. The GFP1-9 transcript has approximately 200 amino acids which yields $\ce{k_\text{MG}=4.5}$
    min<sup>-1</sup>. </p>

  <p>d) Maturation times can range from a couple of minutes up to an hour for GFPs in <i>E. coli</i> [3]. We chose a maturation time of 6 min for GFP1-9.</p>

  <p>e) The rate constants were taken for 17&beta;-estradiol.</p>

  <p>f) </p>

  <p>g) A dose-response scheme for the salicylate inducible promoter was taken from the iGEM parts registry documentation page (see figure 13). In the
    experimental setup, eight doses of salicylate were added every 30 minutes, activating the expression of a GFP. The response was measured as an
    induction ratio of the fluorescence with respect to the uninduced state. Figure 1 illustrates the dose-response curve. In the setup we assume an
    induction base level of 100 nM which corresponds to approximately 100 proteins per cell at the uninduced state. 
  </p>

  <p>h) In a paper from 2013, a tripartite split GFP-association was studied by complementing GFP1-9 with GFP10-SR-GFP11 (SR = sulfite reductase)
    fusion protein [6]. The construct was studied <i>in vitro</i> under equimolar amounts of GFP1-9 and GFP10-SR-GFP11 to yield a time-intensity
    fluorescence plot. We chose to estimate the association and dissociation rate parameters from the experiments carried out. Since the reactions were
    carried out <i>in vitro</i>, we assume that the time frame in which the measurement was carried out (12 hours)  is too short for the proteins to
    decompose. In addition, we assume that once a GFP has been associated, it cannot dissociate back into it's respective fragments (this is reasonable
    as one does not expect a GFP under mild conditions to spontaneously dissociate into three parts). Also assuming that no side reactions occur and
    that everything that can react eventually reacts, one can derive the following relation (see appendix) for the concentration $c(t)$ of the
    complemented protein,

    $$c(t) = \begin{cases} G_0  - \frac{G_0}{1 + G_0kt}, & \mbox{if } G_0=E_0 \\
    \frac{G_0E_0 - G_0E_0e^{k(G_0-E_0)t}}{E_0 - G_0e^{k(G_0-E_0)t}}, & \mbox{if } G_0\ne E_0 \end{cases} \tag{1}$$

    where $G_0$ and $E_0$ are the concentrations of GFP1-9 and GFP10-ER-GFP11 at time $t=0$. At figure 3 in the article, a steady state defined as 95 %
    of the asymptotic concentration is reached after approximately 9 hours. As equimolar amounts of proteins were used (3.5 &micro;M), we get the expression

    $$0.95G_0 = G_0  - \frac{G_0}{1 + 9G_0k}\tag{2}$$

    which yields $\ce{k_\text{aGFP}=1.0 \cdot 10^{-5}}$ nM<sup>-1</sup>min<sup>-1</sup>.

  <p>i) The constant was estimated from the permeability $p$ using the relation $k = p A / V$ where $A$ is the surface area and $V$ the cell volume.
    We assume that the cell is spherical, yielding the constant $\ce{k_\text{dS}=0.281}$ min<sup>-1</sup> from the permeability $\ce{p=9 \cdot 10^{-8}}$
    cm s<sup>-1</sup> [3].</p>

  <p>j) The constant was estimated from the diffusion constant $D$ using the relation $k = D/A$ where $A$ is the surface area. We assume that
    the cell is spherical, yielding $\ce{k_\text{dE}=1.44}$ min<sup>-1</sup> from the diffusion constant $\ce{D=10^{7}}$ Å<sup>2</sup>/s. [4]</p>


  <p>k) We set the lifetime of the mRNAs to 10 min, which yields the degradation rate $\lambda=0.2$ [3].</p>

  <p>l) We set the lifetime of NahR to 2 hours.</p>

  <p>m)  The half-life for the estrogen receptor ligand binding domain in mammalian cells has been shown to be more than 3 hours in the unbound state
     and 1-3 hours for the ligand bound state. Usually, protein lifetimes in <i>E. coli</i> cells vary a lot and in the absence of information and
     experimental data we chose a life time of 3 hours for both the ligand bound and unbound state.</p>





<br>
<br>
<br>
<br>
<br>





  <p>h)  Half lifes for common fluorescent proteins has been shown to range over one to two hours [7]. We choose a half-life of 120 min, yielding the rate constant $\lambda = \log(2)/120$ min<sup>-1</sup>.

  <p>i)  The half-life for the estrogen receptor ligand binding domain in mammalian cells has been shown to be more than 3 hours in the unbound state
     and 1-3 hours for the ligand bound state. Usually, protein lifetimes in <i>E. coli</i> cells vary a lot and in the absence of information and
     experimental results we chose a life time of 3 hours for both the ligand bound and unbound state.</p>

  <p>j)  In the bionumbers database, half-lifes for varieties of GFP in <i>E. coli</i> range from a couple of minutes to hours. We choose a half-life of two hours, which yields $ce{\lambda_\text{GFP} = 0.0058}$ [Bionumbers]. This is assumed to be the same for both the GFP1-9 fragment as well as for the ER-&alpha; bound protein.</p>


  <h3>Initial values</h3>
  <p>From the obtained parameters, the initial values for the systems can be calculates as .. -> Operator content framför allt.

  <h3>Math stuff</h3>
    $$\ce{a + b ->[k] c}$$
    $$\ce{a ->[\lambda_1] \phi}$$
    $$\ce{b ->[\lambda_2] \phi}$$
    $$\ce{c ->[\lambda_3] \phi}$$
  where $\text{a}$ is the GFP1-9 fragment and $\text{b}$ the GFP10-linker-GFP11 fragment. The product $\text{c}$ is then the recomplemented GFP. Since the reactions were carried out in vitro, we assume that no degradation occur, that is $\lambda_i = 0$. By balancing the masses we get the following equations:
    $$\frac{dc}{dt} = kab$$
    $$\frac{db}{dt} = -kab$$
    $$\frac{da}{dt} = -kab$$
  These equations can be reduced into one by noticing that $\frac{dc}{dt} + \frac{db}{dt} = 0$ and $\frac{dc}{dt} + \frac{da}{dt} = 0$. We see that the sum of c and b, and c and a is constant. These relations can be solved to yield
    $$b = b_0 - c$$
    $$a = a_0 - c$$
    $$\frac{dc}{dt} = k(a_0-c)(b_0-c) = ka_0b_0 - (a_0 + b_0)c + c^2$$
  Here, $a_0$ and $b_0$ are the initial concentrations of $a$ and $b$. This is a separable differential equation which can be solved by separating the time and concentration terms. One can show (see appendix) that the solution is
  $$c(t) = \begin{cases} a_0  - \frac{a_0}{1 + a_0kt}, & \mbox{if } a_0=b_0 \\
  \frac{a_0b_0 - a_0b_0e^{k(a_0-b_0)t}}{b_0 - a_0e^{k(a_0-b_0)t}}, & \mbox{if } a_0\ne b_0 \end{cases}$$

  In figure xx of the article, it is shown for the optimal GFP fragments that steady state is reached after approximately five hours. Assuming that, that value s 95 % of the asymptotic concentration and that all species will react, we can calculate the parameter $k$ through the following. In the experiment they had equimolar concentrations (3.5 &micro;M) of GFP1-9 and GFP10-SR-GFP11 (SR = sulfite reductase). This yeilds
  $a_0 = b_0$:
  $$0.95a_0  = a_0 - \frac{a_0}{1+a_0kt_{0.95}}$$
  $\Longrightarrow k = 1.8\cdot 10^{-5}$


  <h3>References</h3>
  <p>[1] Merck Millipore pETDuet-1 docuementation. <a href="http://www.merckmillipore.com/SE/en/product/pETDuet%E2%84%A2-1-DNA---Novagen,EMD_BIO-71146#documentation">Link here</a></p>
  <p>[2] Michaeil Stamatakis and Nikos V. Mantzaris. "Comparision of Determinstic and Stochastic Models of the <i>lac</i> Operon Genetic Network". Biophysical Journal Volume 96. Feb 2009 887-906</p>
  <p>[3] Milo, R., Phillips, R. and Orme, N. (2016). Cell biology by the numbers. New York, NY: Garland Science. </p>
  <p>[4] Rebecca L. Rich*, Lise R. Hoth†, Kieran F. Geoghegan†, Thomas A. Brown†, Peter K. LeMotte†, Samuel P. Simons†,Preston Hensley†, and David G. Myszka*. "Kinetic analysis of estrogen receptor ligand interactions". </p>
  <p>[5] Park, H., Lim, W. and Shin, H. (2005). In vitro binding of purified NahR regulatory protein with promoter Psal. Biochimica et Biophysica Acta (BBA) - General Subjects, 1725(2), pp.247-255.</p>
  <p>[6] Cabantous, S., Nguyen, H., Pedelacq, J., Koraïchi, F., Chaudhary, A., Ganguly, K., Lockard, M., Favre, G., Terwilliger, T. and Waldo, G. (2013). A New Protein-Protein Interaction Sensor Based on Tripartite Split-GFP Association. Scientific Reports, 3(1).</p>
  <p>[7] John Gutknecht. "Aspirin, acetaminophen and proton transport through phospholipid bilayers and mitochondrial membranes". Molecular and Cellular Biochemistry 114: 3-8, 1992</p>
  <p>[8] Idit Oren, Sarel J. Fleishman, Amit Kessel, Nir Ben-Tal. "Free Diffusion of Steroid Hormones Across Biomembranes: A Simplex Search with Implicit Solvent Model Calculations". Biophysical Journal Vol. 87, August 2004, p. 768-779.</p>
  <p>[9] Kocanova, S., Mazaheri, M., Caze-Subra, S. and Bystricky, K. (2010). Ligands specify estrogen receptor alpha nuclear localization and degradation. BMC Cell Biology, 11(1), p.98.</p>
  <p>[Bionumbers] The bionumbers databse. Milo et al. Nucl. Acids Res. (2010) 38 (suppl 1): D750-D753
</div>
</div>
  </div>
</div>
</section>
</html>
{{Lund/footer}}
