{{Lund}}
<html>
</div> <!-- iGEM stuff-->

<script type="text/javascript">
  $('#navbar').addClass('navbar-transparent');
</script>

<header class="modeling-landing">
  <div class="container">
    <h1 class="display">Modeling</h1>
  </div>
</header>

<section id="modeling">
  <div class="container">
  <nav class="navbar navbar-default navbar-static-top nav nav-pills" style="background-color:white">
    <li><a href="/Team:Lund/Model.html">Results</a></li>
    <li><a href="/Team:Lund/Modeling/ModelDerivation.html">Model derivation</a></li>
    <li class="active"><a href="/Team:Lund/Modeling/ParameterEstimation.html">Parameter estimation</a></li>
    <li><a href="/Team:Lund/Modeling/Implementation.html">Implementation</a></li>
  </nav>
  <h3>Overview</h3>
  <p>In this section we provide the parameters used in the model. Table 1 yields an overview for all parameters with a
  corresponding reference to either the literature or explanation in the description section.</p>
  <br>

  <h3>Parameters</h3>
  <table class="table">
    <caption>Table 1: List of parameters, a) Constant was calculated from the permeability $p$ using the relation $k = p \frac{A}{V}$ where $A$ is the surface area and $V$ cell volume, assuming the cell is a sphere.  b) Constant was calculated from the diffusion constant $D$ via $k = \frac{D}{A}$ where $A$ is the cell surface area, assuming a perfect sphere. c)</caption>
    <thead>
      <tr>
        <th>Parameter</th>
        <th>Value</th>
        <th>Unit</th>
        <th>Description</th>
        <th>Reference</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>$\ce{C_\text{p}}$</td>
        <td>$40$</td>
        <td>-</td>
        <td>Copy number</td>
        <td>[1]</td>
      </tr>
      <tr>
        <td>$\ce{V}$</td>
        <td>$8 \cdot 10^{-16}$</td>
        <td>L</td>
        <td>E. coli cell volume</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{MR}}$</td>
        <td>0.11</td>
        <td>$\text{min}^{-1}$</td>
        <td>MR transcription rate constant</td>
        <td>[2]<sup>[a]</sup></td>
      <tr>
      <tr>
        <td>$\ce{k_\text{R}}$</td>
        <td>15</td>
        <td>$\ce{min}^{-1}$</td>
        <td>R translation rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{2R}}$</td>
        <td>$50$</td>
        <td>$\ce{\text{nM}^{-1}\cdot min^{-1}}$</td>
        <td>R dimerization rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{-2R}}$</td>
        <td>$10^{-3}$</td>
        <td>$\ce{min}^{-1}$</td>
        <td>R$_2$ dissociation rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{r}}$</td>
        <td>$960$</td>
        <td>$\text{nM}^{-1}\cdot \text{min}^{-1}$</td>
        <td>Operator repression rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{-r}}$</td>
        <td>$2.4$</td>
        <td>$\text{min}^{-1}$</td>
        <td>Operator derepression rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{dr1}}$</td>
        <td>$3\cdot 10^{-7}$</td>
        <td>$\text{nM}^{-2}\cdot\text{min}^{-1}$</td>
        <td>Free R$_2$ repression rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{-dr1}}$</td>
        <td>12</td>
        <td>$\text{min}^{-1}$</td>
        <td>Free R$_2$ derepression rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{dr2}}$</td>
        <td>$3\cdot 10^{-7}$</td>
        <td>$\text{nM}^{-2}\cdot \text{min}^{-1}$</td>
        <td>Operator bound R$_2$ repression rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{-dr2}}$</td>
        <td>$4.8 \cdot 10^{3}$</td>
        <td>$\text{nM}^{-1} \cdot \text{min}^{-1}$</td>
        <td>Operator bound R$_2$ derepression rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{MN}}$</td>
        <td>$3.09$</td>
        <td>$\text{min}^{-1}$</td>
        <td>NahR mRNA transcription rate constant</td>
        <td>[7]<sup>[b]</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{ME}}$</td>
        <td>$3.18$</td>
        <td>$\text{min}^{-1}$</td>
        <td>Estrogen receptor LBD mRNA transcription rate constant</td>
        <td>[7]<sup>[b]</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{MG}}$</td>
        <td>$4.72$</td>
        <td>$\text{min}^{-1}$</td>
        <td>GFP1-9 mRNA transcription rate constant</td>
        <td>[7]<sup>[b]</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{N}}$</td>
        <td>$3.0$</td>
        <td>$\text{min}^{-1}$</td>
        <td>NahR translation rate constant</td>
        <td>[7]<sup>[c]</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{E}}$</td>
        <td>$2.86$</td>
        <td>$\text{min}^{-1}$</td>
        <td>Estrogen receptor LBD translation rate constant</td>
        <td>[7]<sup>[c]</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{G}}$</td>
        <td>4.59</td>
        <td>$\text{min}^{-1}$</td>
        <td>GFP1-9 translation rate constant</td>
        <td>[7]<sup>[c]</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{mG}}$</td>
        <td>$0.167$</td>
        <td>$\text{min}^{-1}$</td>
        <td>GFP1-9 maturation rate constant</td>
        <td>[7]<sup>[d]</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{aEEs}}$</td>
        <td>$7.8 \cdot 10^{16}$</td>
        <td>$\text{nM}\cdot \text{min}^{-1}$</td>
        <td>Estrogen receptor-ligand association rate constant</td>
        <td>[6]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{dEEs}}$</td>
        <td>$7.2 \cdot 10^{-4}$</td>
        <td>$\text{min}^{-1}$</td>
        <td>Estrogen receptor-ligand dissociation rate constant</td>
        <td>[6]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{aNP}}$</td>
        <td></td>
        <td></td>
        <td>NahR-Psal association rate constant</td>
        <td>Estimated from literature data<sup>[e]</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{dNP}}$</td>
        <td></td>
        <td></td>
        <td>NahR-Psal dissociation rate constant</td>
        <td>Estimated from literature data<sup>[e]</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{aSNP}}$</td>
        <td></td>
        <td></td>
        <td>Salicylate-[NahR-Psal] association rate constant</td>
        <td>Estimated from literature data<sup>[e]</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{dSNP}}$</td>
        <td></td>
        <td></td>
        <td>Salicylate-[NahR-Psal] dissociation rate constant</td>
        <td>Estimated from literature data<sup>[e]</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{aGFP}}$</td>
        <td></td>
        <td></td>
        <td>Tripartite GFP association rate constant</td>
        <td>[]<sup>[d]</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{dGFP}}$</td>
        <td></td>
        <td></td>
        <td>Tripartite GFP dissociation rate constant</td>
        <td>[]<sup>[d]</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{dI}}$</td>
        <td>0.92</td>
        <td>$\text{min}^{-1}$</td>
        <td>IPTG diffusion constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{dS}}$</td>
        <td>$0.281$</td>
        <td>$\text{min}^{-1}$</td>
        <td>Salicylate diffusion rate constant</td>
        <td>[3]<sup>[f]</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{dE}}$</td>
        <td>$1.44$</td>
        <td>$\text{min}^{-1}$</td>
        <td>Estrogen diffusion rate constant</td>
        <td>[4]<sup>[g]</sup></td>
      </tr>
      <tr>
        <td>$\ce{\lambda_\text{MR}}$</td>
        <td>$0.462$</td>
        <td>$\ce{min^{-1}}$</td>
        <td>MR degradation rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{\lambda_\text{MN}}$</td>
        <td>$0.2$</td>
        <td>$\text{min}^{-1}$</td>
        <td>NahR mRNA degradation rate constant</td>
        <td>[7]<sup>[g]</sup></td>
      </tr>
      <tr>
        <td>$\ce{\lambda_\text{ME}}$</td>
        <td>$0.2$</td>
        <td>$\text{min}^{-1}$</td>
        <td>Estrogen receptor LBD mRNA degradation rate constant</td>
        <td>[7]<sup>[g]</sup></td>
      </tr>
      <tr>
        <td>$\ce{\lambda_\text{MG}}$</td>
        <td>0.2</td>
        <td>$\text{min}^{-1}$</td>
        <td>GFP1-9 mRNA degradation rate constant</td>
        <td>[7]<sup>[g]</sup></td>
      </tr>
      <tr>
        <td>$\ce{\lambda_\text{R}}$</td>
        <td>0.2</td>
        <td>$\ce{min}^{-1}$</td>
        <td>R degradation rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{\lambda_{\text{R}_2}}$</td>
        <td>0.2</td>
        <td>$\ce{min}^{-1}$</td>
        <td>R$_2$ degradation rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{\lambda_{\text{I}_2\text{R}_2}}$</td>
        <td>0.2</td>
        <td>$\ce{min}^{-1}$</td>
        <td>I$_2$R$_2$ degradation rate constant</td>
        <td>[2]</td>
      </tr>
      <tr>
        <td>$\ce{\lambda_\text{N}}$</td>
        <td></td>
        <td></td>
        <td>NahR degradation rate constant</td>
        <td></td>
      </tr>
      <tr>
        <td>$\ce{\lambda_\text{E}}$</td>
        <td>$2.8 \cdot 10^{-4}$</td>
        <td>$\text{min}^{-1}$</td>
        <td>Estrogen receptor LBD degradation rate constant</td>
        <td></td>
      </tr>
      <tr>
        <td>$\ce{\lambda_\text{G}}$</td>
        <td>$0.0058$</td>
        <td>$\text{min}^{-1}$</td>
        <td>GFP1-9 degradation rate constant</td>
        <td>[]</td>
      </tr>
      <tr>
        <td>$\ce{\lambda_\text{GFP}}$</td>
        <td></td>
        <td></td>
        <td>GFP degradation rate constant</td>
        <td>[]</td>
      </tr>
      <tr>
        <td>$\ce{k_\text{transcription}}$</td>
        <td>$50$</td>
        <td>$\text{nucleotides}\cdot\text{s}^{-1}$</td>
        <td>Average transcription rate</td>
        <td>[7]<sup>[a]</sup></td>
      </tr>
      <tr>
        <td>$\ce{k_\text{translation}}$</td>
        <td>$15$</td>
        <td>$\text{amino acids}\cdot\text{s}^{-1}$</td>
        <td>Average translation rate</td>
        <td>[7]<sup>[a]</sup></td>
      </tr>
    </tbody>
  </table>
  <br>
  <h3>Description</h3>
  <p>[a]  In the referenced article, the original rate constant is 0.23 nM/min for an operator content of 2.08 nM. This yields the rate constant
  $\ce{k_\text{MR} = 0.11}$ min<sup>-1</sup></p>

  <p>[b]  The transcription rate is on average 50 nucleotides per second in <i>E. coli</i> [7]. The NahR transcript has approximately 900 nucleotides which
    yields $\ce{k_\text{MN}=3.33}$ min<sup>-1</sup>. The GFP10-linker-ER-&alpha;LBD-linker-GFP11 transcript has approximately 1800 nucleotides which
    yields $\ce{k_\text{ME}=1.67}$ min<sup>-1</sup>. The GFP1-9 transcript has approximately 600 nucleotides which yields $\ce{k_\text{MG} = 5}$
    min<sup>-1</sup>. </p>

  <p>[c]  The translation rate is on average 15 amino acids per second in <i>E. coli</i> [7]. NahR has approximately 300 amino acids which
    yields $\ce{k_\text{MN}=3}$ min<sup>-1</sup>. GFP10-linker-ER-&alpha;LBD-linker-GFP11 has approximately 600 amino acids which
    yields $\ce{k_\text{ME}=1.5}$ min<sup>-1</sup>. The GFP1-9 transcript has approximately 200 amino acids which yields $\ce{k_\text{MG}=4.5}$
    min<sup>-1</sup>. </p>

  <p>[d]  Antag irreversibel. Man förväntar sig knappast att ett vanligt GFP ska dissociera i två fragment. Och om det gör det så är det genom nedbrytning.</p>


  <p>[e]  Nedbrytningshastigheter väljs så att modellen blir rimlig. Runt sex timmar är vanligt för proteiner? [källa]</p>


  <p>[f]  The constant was estimated from the permability $p$ using the relation $k = p \frac{A}{V}$ where $A$ is the surface area and $V$ the cell
    volume. We assume that the cell is spherical with volume $\ce{V=8 \cdot 10^{-16}}$ L, yielding the constant $\ce{k_\text{dS}=0.281}$ min<sup>-1</sup> from $\ce{p=9 \cdot 10^{-8}}$ cm s<sup>-1</sup> [3].</p>

  <p>[g]  The constant was estimated from the diffusion constant $D$ using the relation $k = \frac{D}{A}$ where $A$ is the surface area. We assume that
    the cell is spherical with volume $\ce{V=8\cdot 10^{-16}}$ L, yielding $\ce{k_\text{dE}=1.44}$ min<sup>-1</sup> from $\ce{D=10^{7}}$ Å<sup>2</sup>/s. [4]</p>

  <p>[h]  Half lifes for common fluorescent proteins has been shown to range over one to two hours [7]. We choose a half-life of 120 min, yielding the rate constant $\lambda = \log(2)/120$ min<sup>-1</sup>.


  <br>
  <br>
  <br>
  <br>
  <br>
  <br>
  <p>
    The transcription rate is on average 50 nucleotides per second (stochastic processes in cell biology, cell biology by numbers).
    <br>
    The translation rate is on average 15 amino acids per second (<a href="http://book.bionumbers.org/what-is-faster-transcription-or-translation/">Cell Biology By numbers</a>).
    <br>
    The average lifetime of mRNA is assumed to be 5 min (<a href="http://book.bionumbers.org/how-fast-do-rnas-and-proteins-degrade/">Cell biology by numbers</a>).
    <br>
    transcription: 50 nucleotides/sec = 3000 nucleotides/min
    translation: 15 amino acids/sec = 900 amino acids/mi  n
    <br>
    NahR: 300 amino acids -> 900 nucleotides -> mRNA: 0.3 min^-1. protein: 0.33 min^-1
    <br>
    GFP10-link-ERa-link-GFP11: 606 amino acids -> 1818 nucleotides -> mRNA 0.606 min^-1 -> 0.673 min^-1
    <br>
    Maturation time sfGFP 6min <a href="http://book.bionumbers.org/what-is-the-maturation-time-for-fluorescent-proteins/">ref</a>
    <br>
    Half-life sgGFP: 28 min - 119 min for R-MCherry GFP. We assume a half-life of 120 min for our sfGFP. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4713137/pdf/360.pdf">link</a>.
    <br>
    Half lifes for common fluorescent proteins has been shown to range over one to two hours [<a href="http://bionumbers.hms.harvard.edu/search.aspx?log=y&task=searchbytrmorg&trm=half-life+gfp+Bacteria+Escherichia+coli">bionumbers</a>]. We take a value of 120 minutes.
    -> $\lambda_\text{G} = \log{(2)}/120 \text{min}^{-1}$
    <br>
    For the estrogen receptor alpha, the half-life varies from 2 to 24 hours in human (link). We assume a half-life of 6 hours.
    <br>
    The folding times for the proteins are estimated as directly proportional to the sequence length via the relation $N^{1/2} = \log(\tau)$, where N is the chain length and $\tau$ is the folding time in microseconds. [källa].
    <br>
    For membrane diffusion rates estimated from the permeability $p$, we estimate the rate constants as $\text{k} = p\frac{A}{V}$.
    <br>
    The NahR rate constants were obtained by fitting our system of equations to induction ratios taken from literature [link], <b>assuming</b> an expression of 100 nM at base level and a steady state after 4 hours.
  </p>

  <h3>References</h3>
  <p>[1] Merck Millipore pETDuet-1 docuementation. <a href="http://www.merckmillipore.com/SE/en/product/pETDuet%E2%84%A2-1-DNA---Novagen,EMD_BIO-71146#documentation">Link here</a></p>
  <p>[2] Michaeil Stamatakis and Nikos V. Mantzaris. "Comparision of Determinstic and Stochastic Models of the <i>lac</i> Operon Genetic Network". Biophysical Journal Volume 96. Feb 2009 887-906</p>
  <p>[2] http://book.bionumbers.org/how-big-is-an-e-coli-cell-and-what-is-its-mass/ </p>
  <p>[3] John Gutknecht. "Aspirin, acetaminophen and proton transport through phospholipid bilayers and mitochondrial membranes". Molecular and Cellular Biochemistry 114: 3-8, 1992</p>
  <p>[4] Idit Oren, Sarel J. Fleishman, Amit Kessel, Nir Ben-Tal. "Free Diffusion of Steroid Hormones Across Biomembranes: A Simplex Search with Implicit Solvent Model Calculations". Biophysical Journal Vol. 87, August 2004, p. 768-779.</p>
  <p>[5] Bionumbers average translation and transcription rate.
  <p>[6] Rebecca L. Rich*, Lise R. Hoth†, Kieran F. Geoghegan†, Thomas A. Brown†, Peter K. LeMotte†, Samuel P. Simons†,Preston Hensley†, and David G. Myszka*. "Kinetic analysis of estrogen receptor ligand interactions". </p>
  <p>[7] Milo, R., Phillips, R. and Orme, N. (2016). Cell biology by the numbers. New York, NY: Garland Science. </p>
</div>
</div>
  </div>
</div>
</section>
</html>
{{Lund/footer}}
